packages/client/src/app/components/modals/gacha/sidebar/controls/Controls.tsx

```
@@ -73,6 +73,7 @@ export const Controls = ({
   };
 
   const isButtonVisible = () => {
+    return tab === 'GACHA';
     return tab === 'GACHA' || tab === 'REROLL';
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)